api/v1alpha1/gcpchaos_types.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 Chaos Mesh Authors.
```

Comment:
Looks like we need to update the template XD

Reply:
Yes, I may have to manually modify this first.